blackbelt/tasks.py

```diff
@@ -9,7 +9,7 @@ class BlackBelt(click.MultiCommand):
     def list_commands(self, ctx):
         rv = []
         for filename in os.listdir(plugin_folder):
-            if filename.endswith('.py'):
+            if filename.endswith('.py') and filename is not '__init__.py':
```

Comment:
Unlike in CS, `is` [is identity operator in Python](http://stackoverflow.com/a/13650309/325365), so you want to use plain oldschool `!=` instead.